contracts/script/WriteToContractsRegistry.s.sol

```diff
@@ -0,0 +1,54 @@
+// SPDX-License-Identifier: UNLICENSED
```

Comment:
actually did this to make the scripts more clean, otherwise it ends up a big script which is a difficult process for a new contributor

Reply:
Maybe adding this as a lib to `utils/` would reduce the clutter

it would be a stateless library along the lines of

```
deploy() returns (RegistryDeploymentLib.DeploymentData)

writeOutput(RegistryDeploymentLib.DeploymentData)

writeToRegistry(CoreDeploymentLib.DeploymentData, ICSDeploymentLib.DeploymentData)
```

And then it would add about 3 lines to the deploy script and would be useful to have in the tests where we just wouldn't write the json

Reply:
the struct is a bit overkill in this scenario since it's just a single address but i wanted to keep with the pattern from the other deployment libraries